src/rave.vue

```diff
@@ -1,142 +1,147 @@
 <template>
-  <button
-    :class="``"
-    @click="payWithRave"
-  >
+  <button :class="``" @click="payWithRave">
     <slot>Make Payment</slot>
   </button>
 </template>
```

Comment:
Did you edit anything in this file or you re-format the code?

Reply:
hello @iamraphson, yes I edited the code in that file and in the readme as well